Assets/MRTK/Examples/Experimental/Joystick/JoystickSliders.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Joystick
+{
+    /// <summary>
+    /// Example script to demonstrate adding sliders to control the joystick values at runtime.
+    /// </summary>
+    public class JoystickSliders : MonoBehaviour
+    {
+        public PinchSlider SliderMove;
+        public PinchSlider SliderScale;
+        public PinchSlider SliderRotate;
+        public PinchSlider SliderRebound;
+        public PinchSlider SliderSensitivityHorizontal;
+        public PinchSlider SliderSensitivityVertical;
+        //
+        public TextMeshPro TextMove;
+        public TextMeshPro TextScale;
+        public TextMeshPro TextRotate;
+        public TextMeshPro TextRebound;
+        public TextMeshPro TextSensitivityHorizontal;
+        public TextMeshPro TextSensitivityVertical;
+        //
+        public JoystickController[] Joysticks;
+        //
```

Comment:
```suggestion

```

Reply:
(Same type of comment for other emtpy comments below that are empty and are there to add vertical space.